.github/pull_request_template.md

```diff
@@ -0,0 +1,87 @@
+# Pull Request Title
+## Fixes \#(your issue number)
+
+<br>
+
+# Change Description
+<em>
+Please write a summary of the change and specify which Issue this fixes. This should cover the motivation for making the change and provide almost enough context to review the change in isolation. While the issue description is a good place to discuss the failure/requirements, this is the ideal space for explaining why this design fixes the issue and why you designed it the way you did.
+
+<br>
+
+- If there are important design decisions, particularly those that leverage functionality outside this change or were caused by existing limitations, they should be noted here.
+- If there is a design specification which relates to this PR, please provide a link to it here.
+- If there are any dependencies (third-party or internal downstream) which are added or impacted, please note them here. And if their limitations informed your design, please note how they did so here (particularly if this information was not captured in a design specification).
+</em>
+
+Text goes here
```

Comment:
I would suggest to encourage in the template a list of checklist items. E.g.

It is useful to provide a [task list](https://docs.github.com/en/get-started/writing-on-github/working-with-advanced-formatting/about-task-lists) within the pull request. These should breakdown the overall PR into units of work which quite often can be associated to either a specific issue (if the PR closes multiple) or a commit. The task list allows the developer to update on development progress while the PR is in draft. It provides insight into the current state of the PR at any specific point in time. Once the PR is complete (ready for review) it is expected that each item would be checked or justification provided as to why the task will not be completed in this PR (perhaps as it has moved to a new issue). An example task list:

 - [ ] Define a test
 - [ ] Implement working version
 - [ ] Document feature by providing worked example
 - [ ] Update README

Reply:
I have a pre-PR checklist but agree this task this serves a special purpose. Given the ability to provide at-a-glance development transparency, I'm going to put this at the very top under the 'Fixes Issue Number' line